src/applications/coronovirus-screener/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "COVID-19 Screener",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "covid19screener",
+  "rootUrl": "/covid19screener",
+  "production": false
```

Comment:
You may want to update `src/applications/registry.json` with `vagovprod` set to false for this application.

Reply:
thanks! any idea who wins if there is a conflict b/w registry.json and this manifest file?

Reply:
does our app need an entry in `src/applications/registry.json`? it's not mentioned in [yeoman docs](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/generator/) i see it in [manual setup](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/new-application) though

Reply:
Right now the `registry.json` file is just used to [create the react landing pages](https://github.com/department-of-veterans-affairs/vets-website/blob/eccb647865b3d827f128f1392093b7a28771768f/src/site/stages/build/plugins/create-react-pages.js#L3-L24) for each application. One isn't overriding the other, so there shouldn't be any conflicts.